### DISCUSSION TITLE
Contentment on Umbraco 14

### DISCUSSION THREAD

**Discussion by sebbys83** [leekelleher/umbraco-contentment#402] [Question]
Hi @leekelleher ,

First of, I'm a massive fan of Contentment and soon as I noticed there's Contentment for Umbraco 14, I thought I need to try it. I'm experiencing an issue with installing contentment through nuget tho. Seems like there's a package issue (Umbraco.Cms.Web.Website)

Could you help ?

Version conflict detected for Umbraco.Cms.Web.Website. Install/reference Umbraco.Cms.Web.Website 14.0.0 directly to project ThisBeCool.Web.Cms to resolve this issue.

NU1107: Version conflict detected for Umbraco.Cms.Web.Website. Install/reference Umbraco.Cms.Web.Website 14.0.0 directly to project ThisBeCool.Web.Cms to resolve this issue. 
 ThisBeCool.Web.Cms -> Umbraco.Cms 14.0.0 -> Umbraco.Cms.Targets 14.0.0 -> Umbraco.Cms.StaticAssets 14.0.0 -> Umbraco.Cms.Web.Website (>= 14.0.0 && < 15.0.0) 
 ThisBeCool.Web.Cms -> Umbraco.Community.Contentment 5.0.0 -> Umbraco.Cms.Web.Website (>= 13.2.0 && < 14.0.0).
Package restore failed. Rolling back package changes for 'ThisBeCool.Web.Cms'.

**Comment by leekelleher** [comment]
Hi @sebbys83,

Contentment for Umbraco 14 (Bellissima) is currently under active development and hasn't been released yet. The latest status updates are here: https://github.com/leekelleher/umbraco-contentment/discussions/357.

The current release on NuGet (v5.0.0) will only target Umbraco v13. I'm hoping to get an alpha version (`v6.0.0-alpha001` for v14) out within the next week. But keep in mind that it's a complete rewrite of all the UI and wouldn't be considered "production-ready" until it's out of alpha.

**Reply by sebbys83** [reply]
@leekelleher ,

I got over excited when I saw the post and wanted to try it on my personal project. I guess if I read the post (which I did just now), I wouldn't have bothered you.

Yeah, it's a whole new UI and I can imagine there's a lot of effort to actually upgrade it and make it work. I've been looking into developing a backoffice content app (only for my project) and it's like moving from .NET framework to .Net Core :) (or from knockout to angular.

If you need another pair of hands to help with the project, feel free to share it and I'll see how I could assist you
